roles/inspec/templates/etc/inspec/host-controls/controls/host.rb

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
+# license: Apache 2
+#title 'host-controls'
+
+require_controls 'inspec-openstack-security' do
+# BEGIN ANSIBLE MANAGED BLOCK cinder
```

Comment:
I don't believe you need to prefill these, the block in file should create them.